pokemongo_bot/walkers/polyline_walker.py

```diff
@@ -20,10 +19,3 @@ def __init__(self, bot, dest_lat, dest_lng):
         self.pol_alt = self.polyline.get_alt() or self.actual_alt
         super(PolylineWalker, self).__init__(self.bot, self.pol_lat, self.pol_lon,
                                              self.pol_alt, fixed_speed=self.speed)
-
```

Comment:
I tested that.. the StepWalker returns

`return distance(self._new_position[0], self._new_position[1], self.destLat, self.destLng) < 1`

will still always be True for the polyline....